hydra-node/test/Hydra/HeadLogicSpec.hs

```diff
@@ -670,6 +672,34 @@ spec =
               } -> null localTxs
           _ -> False
 
+    prop "empty inputs in decommit tx are prevented" $ \tx -> do
+      let ledger = cardanoLedger Fixture.defaultGlobals Fixture.defaultLedgerEnv
+      let st =
+            Open
+              OpenState
+                { parameters = HeadParameters defaultContestationPeriod threeParties
+                , coordinatedHeadState =
+                    CoordinatedHeadState
+                      { localUTxO = mempty
+                      , allTxs = mempty
+                      , localTxs = []
+                      , confirmedSnapshot = InitialSnapshot testHeadId mempty
+                      , seenSnapshot = NoSeenSnapshot
+                      , decommitTx = Nothing
+                      , version = 0
+                      }
+                , chainState = Prelude.error "should not be used"
+                , headId = testHeadId
+                , headSeed = testHeadSeed
+                , currentSlot = ChainSlot 1
+                }
+
+      let tx' = fromLedgerTx (toLedgerTx tx & bodyTxL . outputsTxBodyL .~ mempty)
```

Comment:
This setting the outputs to empty. The test description says empty inputs are not allowed. Either set inputs to empty or change the description.